### PR TITLE
Respond with correct message if unsupported file is loading.

### PR DESCRIPTION
**Pull request by kegor**
**Problem:**
We need to message user about currently unsupported file format to load/open.

**Solution:**
1. Api version of the Editor library was updated (replicated solution from https://github.com/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pull/8).
2. Was added logic for responding with useful message in case of attempt to open file with currently unsupported format.

**Screenshots:**
_Unsupported file format (PDF):_
unsupported_format

_Password protected file (Excel):_
password_protected_excel

_Incorrect password (Excel):_
incorrect_password_excel

### CODE REVIEW

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/GroupDocs.Total-for-NET-MVC/GroupDocs.Total-for-NET-MVC/pullRequest?prid=4605782&bid=15453651)

File: src/Products/Editor/Controllers/EditorApiController.cs [hunk 1]
```diff
@@ -203,6 +214,12 @@ public HttpResponseMessage UploadDocument()
                 }
                 else
                 {
+                    var options = EditorHandler.DetectOptionsFromExtension(url);
+                    if (options == null)
+                    {
+                        throw new Exception("Unsupported format.");
```
Comment (egorovpavel) [inline src/Products/Editor/Controllers/EditorApiController.cs]: Codacy Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/app/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=4605782)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=4605782&bid=15453651)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=4605782&bid=15453651)

**Comment by kegor** [comment]
Exceptions were checked and now awaiting for further instructions.

**Comment by kegor** [comment]
Currently we already have desired behavior on opening the file with currently unsupported format in master branch (Exception message generates in the Editor library), thus I think we can close this PR:

image

**Pull request closed without merging**